phases/account.go

```diff
@@ -1,9 +1,106 @@
 package phases
 
-import "fmt"
+import (
+	"bufio"
+	"encoding/json"
+	"fmt"
+	"os"
+	"strconv"
+	"strings"
+	"net/http"
 
-// Account ...
-func Account() (string, error) {
-	fmt.Println("SetAccount")
-	return "", nil
+	"github.com/bitrise-io/go-utils/log"
+)
+
+type organizationData struct{
+	Name string
+	Slug string
+}
+
+func isValid(choice int, limit int ) bool {
+	return choice >= 1 && choice <= limit
+}
+
+// Account returns the slug of the selected account. If the user selects
+// the personal account, the slug is empty.
+func Account(apiToken string) (string, error) {
+	req, err := http.NewRequest(http.MethodGet, "https://api.bitrise.io/v0.1/organizations", nil)
+	if err != nil {
+		return "", err
+	}
+
+	req.Header.Set("Authorization", "token " + apiToken)
+	res, err := http.DefaultClient.Do(req)
+	if err != nil {
+		return "", err
+	}
+
+	if res.StatusCode != 200 {
+		return "", fmt.Errorf("fetch orgs: %s", res.Status)
+	}
+
+	m := struct{
+		Data []organizationData
+	}{}
+	if err := json.NewDecoder(res.Body).Decode(&m); err != nil {
+		return "", err
+	}
+	
+	req, err = http.NewRequest(http.MethodGet, "https://api.bitrise.io/v0.1/me", nil)
```

Comment:
NOTE: if we ask the user to use the authenticated user instead of any orgs then this call could be saved. might worth to ask someone if it is required to ask the name exactly or can be just a selection of personal account

Reply:
True, but if the user has more than one personal accounts it can be helpful to see which one is going to be used. I discussed with the product team and it was decided that this should remain as is.